docs/observability/aws/faq.md

```diff
@@ -52,12 +71,18 @@ Any Sumo Logic customer can leverage Sumo Logic AWS Observability solution fea
 
 ### Is there a demo of the solution?
 
-See the [Sumo Logic AWS Observability demo video](https://www.sumologic.com/video/aws-observability-demo/)
-for a demo of the solution.
-
-![aws-demo-video.jpg](/img/observability/aws-demo-video.jpeg)
-
-
+See the [Sumo Logic AWS Observability demo video](https://www.sumologic.com/video/aws-observability-demo/) for a demo of the solution.
```

Comment:
I'd remove one of the video links, otherwise it introduces confusion

```suggestion
See the Sumo Logic AWS Observability video below for a demo of the solution.
```

Reply:
Done, @kimsauce.